plugins/essif-lab_contactform7/src/Application/activator.php

```diff
@@ -1,9 +1,13 @@
 <?php
 
+namespace TNO\ContactForm7\Application;
+
+use TNO\ContactForm7\Implementation\Essif_Lab_Contactform7_Logic;
+
 class Essif_Lab_contactform7_Activator {
```

Comment:
Naamgeving is incorrect, volgens mij vereist PSR4 het ook om de klasnamen gelijk te houden aan de filename. Dus deze class moet hernoemt worden naar "activator". Echter wordt daarnaast in ons project (upper) camelcase gehanteerd dus zou je deze klas en dus ook file moeten hernoemen naar "Activator".